web/src/app/core/resolvers/public-projects.resolver.ts

```diff
@@ -0,0 +1,38 @@
+// Core modules
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, Resolve } from '@angular/router';
+
+// Third party modules
+import { Observable } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+// Dashboard hub model and services
+import { ProjectService } from '@core/services/index.service';
+import { IProject } from '@shared/models/index.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PublicProjectResolver implements Resolve<IProject[]> {
+
+  /**
+   * Life cycle method
+   * @param fns AngularFireFunctions
```

Comment:
We don't have these params in constructor so we should remove them here from comment too

Reply:
I will fix it